projects/nginx/conf_fuzzer.cc

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016 Google Inc.
```

Comment:
It is ideal to move the fuzzer in own repository, see https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md. For now, it is ok.